src/cmakeTools.ts

```diff
@@ -1123,22 +1108,19 @@ export class CMakeTools implements vscode.Disposable, api.CMakeToolsAPI {
      */
     static async createForDirectory(folder: vscode.WorkspaceFolder, ext: vscode.ExtensionContext): Promise<CMakeTools> {
         // Create a context for the directory
-        const dir_ctx = DirectoryContext.createForDirectory(folder, new StateManager(ext, folder));
-        return CMakeTools.create(ext, dir_ctx);
+        const dirContext = DirectoryContext.createForDirectory(folder, new StateManager(ext, folder));
+        return CMakeTools.create(ext, dirContext);
     }
 
-    private _activeKit: Kit | null = null;
-    get activeKit(): Kit | null {
```

Comment:
It's ok to keep this `get`.  The cases we wanted to remove are when there's a trivial `get` and `set`.

Reply:
I don't know if we want to allow other modules to set the activeKit